### ISSUE TITLE
Mettre en place JSHint

### ISSUE THREAD

**Issue by camillebaronnet** [ArdechDromDev/website#16]

**Comment by samsalas** [comment]
Quid de CSSLint ?
J'avais réalisé un job jenkins basé sur phantomJS qui lance JSSHint, CSSHint et donne un score. Mais c'est un peu trop complexe, faut un jenkins, etc...
Par contre, on peut juste demander à chaque reviewer de passer le JS sous jshint.org et csslint.net. Et de tracer la réponse dans la review. En attendant mieux...

**Comment by mbaechler** [comment]
Ou bien on utilise un outil de CI en ligne

**Comment by samsalas** [comment]
Oui, je me demandais si netlify le faisais... @cbouvat @vinzcoco vous savez ???

**Comment by cbouvat** [comment]
@samsalas je n'ai pas regardé, mais je pense qu'il faut plutôt utiliser un outil externe en le plug sur ce dépôt ?!?
